### DISCUSSION TITLE
Help, docx and odt is just displayed as weird gibberish

### DISCUSSION THREAD

**Discussion by Reael** [WinMerge/winmerge#2181] [Q&A]
So I transferred a ton of old documents to my new computer and I have several copies of the same files, some are up to date, some are outdated. All I wanted to do is compare these files so I can get rid of the outdated files. Loading two files (basic Latin alphabet) into winmerge just gives me this gibberish and I have no idea what to do about it.
![gibberish](https://github.com/WinMerge/winmerge/assets/19366611/db6bc973-17b1-4e49-b8c9-196a58f89450)
I installed the program with all plugins, I downloaded a docx plugin separately, I uninstalled the whole thing, I reinstalled it with no plugins (I just assumed it might be to some code-related plugin), I changed a couple of options (I disabled 'detecting codepage' and other things) but this is all I get. Googling didn't help me much. I'm confused because it's just plain Latin, it's not Hebrew or Farsi or any language with signs. Despite maybe äöü and I'm used to programs just displaying a weird symbol instead of äöü but the whole files aren't readable, so I guess it's due to the docx and odt files, as it loads txt just fine.

**Comment by sdottaka** [comment]
If you have MS Word installed on your PC, you can use the CompareMSWordFiles plugin as shown below to extract and compare the text in your documents.

![image](https://github.com/WinMerge/winmerge/assets/98126/5fa0558a-c553-4bed-ba3e-bf1f2687283c)

If MS Word is not installed, please use the Apache Tika plugin as below. However, Java must be installed and the first time will cause its download of tika.jar.

![image](https://github.com/WinMerge/winmerge/assets/98126/e2c61ac3-5794-46c3-84db-412fe642d95a)
